evaluator.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ga4gh
+
+import (
+	"context"
+	"errors"
+	"fmt"
+)
+
+// Evaluator combines both parsing and validation of authorization tokens.
+type Evaluator struct {
```

Comment:
OK, I still think this package is poorly named and we really want something like identity.Evaluator, etc.  From a user point of view, you'll have things like ga4gh.Evaluator and ga4gh.Shim which lack the important context in their name.  I'd be less picky about it except this is going to end up as a public API.

However, I'll leave it up to the maintainers to decide what to do about it since it won't be you or I.